app/errors/command_error.rb

```diff
@@ -1,6 +1,23 @@
 class CommandError < StandardError
   attr_reader :code, :error_details
 
+  def self.with_error_handling(&block)
+    block.call
+  rescue GdsApi::HTTPServerError => e
+    should_suppress = (PublishingAPI.swallow_draft_connection_errors && e.code == 502)
```

Comment:
The existing code only suppresses errors in draft, not live; this does both.

Although I'm wondering how much we care. We originally had this before the draft content store was running in production. Nowadays it is only useful for preventing errors in dev, where you typically only run one content store at a time; maybe we should be a bit more explicit that this is what it is doing.